SimpleBlockX4.md

```diff
@@ -0,0 +1,131 @@
+    * For command line type `volume all size auto factor 5`
+      - This command creates an auto sized mesh with the default/medium coarseness
+      - Typing `volume all size #` would select the first four volumes as listed in the model tree and set the size of any future mesh to # units.
+    * Note that one could select smaller subsets of volumes and mesh each at different levels of coarseness. This may be useful if some volumes require more/less precision than others.
+2. Mesh files with tetmesh
+    * In the command panel directly to the right of the "Intervals" action select the "Mesh" action. Type `all` into the box under "Select Volumes" or click the box and hold the control key while selecting all of the cubes. Then select "Tetmesh" from the dropdown menu under "Select Meshing Scheme" and click mesh.
+    * To mesh through command line you must set the mesh scheme to Tetmesh and then mesh separately. To set the scheme to Tetmesh type "`volume all scheme tetmesh` then after that command is complete simply type "mesh volume all" and the cubes will be successfully meshed with Tetmesh.
+    * You may want to explore how the options for this action affect the final mesh, but default settings are fine for our purposes. Now all surfaces are described by triangular meshes that will be used during analysis.
+3. Follow the steps laid out in the link listed in this description.
+
+#### Assigning Materials in Cubit/Trelis
+_Most commands for this section are best implemented through the command line. More general instructions can
+be found_ [here](http://svalinn.github.io/DAGMC/usersguide/codes/dag-mcnp.html).
+
+  1. The general command for assigning Materials/Densities in Cubit/Trelis
+  `group "mat:[assigned material number]/rho:[assigned density]" add
+  vol [volume numbers]` where the density is in [atoms/barn-cm] for positive values and [g/cc] for negative values.
+  2. The blocks will be given two different materials and then the void and the graveyard volumes will be assigned. These material assignments will depend on the material library you create/use for the project.
+      * To assign one block to the first material use `group "mat:1/rho:0.0223" add vol 1`
+      * To assign the rest of the blocks to the second material use `group "mat:2/rho:0.03" add vol 2 to 4`
+      * To assign the void volume to vacuum use "group `mat:Vacuum" add vol 5`
+      * Finally to assign the graveyard volume use "group `mat:Graveyard" add vol 6`
+
+#### Exporting file to .h5m format
+_This plugin translates the CAD file to something that can be analyzed by DAGMC. Instructions as to how to install this plugin can be found_ [here](https://svalinn.github.io/DAGMC/install/plugin.html).
+
+1. Once the plugin is correctly installed you should receive the message `Loaded Svalinn plugin`, line break, `DAGMC export command available` in the command line.

```

Comment:
installation has to occur before you can open trelis/cubit. the way this is written would make it seem like I can install the plugin while working in trelis and then use it. One has to reopen trelis to be able to use it. So either a section on making sure the plugin is installed should be moved first in the trelis portion of the tutorial. Or note here that if the plugin is not installed, the file should be saved, exited, and reopened.